docs/get_started/chat_agent.md

```diff
@@ -0,0 +1,206 @@
+# Working with the `ChatAgent` Class
+
+In this tutorial, we will explore the `ChatAgent` class designed for managing conversations of CAMEL Chat Agents. The topics covered include:
+
+1. Introduction to the `ChatAgent` class
+2. Creating a `ChatAgent` instance
+3. Understanding the properties of the `ChatAgent` class
+4. Using the methods of the `ChatAgent` class
+
+## Introduction
+
+The `ChatAgent` class is a central component in the CAMEL chat system. It inherits from the `BaseAgent` class and provides functionality to manage and generate chat conversations using different models.
+
+## Creating a `ChatAgent` Instance
+
+To create a `ChatAgent` instance, you need to provide the following arguments:
+
+- `system_message`: The system message for the chat agent.
+- `model`: (optional) The LLM model to use for generating responses.
+- `model_config`: (optional) Configuration options for the LLM model.
+- `message_window_size`: (optional) The maximum number of previous messages to include in the context window.
+- `output_language`: (optional) The language to be output by the agent.
+- `function_list`: (optional) List of available `OpenAIFunction`.
+
+Here's an example of creating a `ChatAgent` instance:
+
+```python
+from camel.agent import ChatAgent, BaseMessage, ModelType
+
+agent = ChatAgent(
+    system_message=BaseMessage(...),
+    model=ModelType.GPT_3_5_TURBO,
+    message_window_size=5,
+    output_language="en"
+)
+```
+
+## Understanding the Properties of the `ChatAgent` Class
+
```

Comment:
It will be better to add properties' type in this introduction.

Reply:
> It will be better to add properties' type in this introduction.

done